pom.xml

```diff
@@ -83,6 +83,11 @@
             <artifactId>junit-jupiter-api</artifactId>
             <version>5.4.2</version>
         </dependency>
+        <dependency>
```

Comment:
Do we still need this dependency/

Reply:
Personaally, I added the smack-legacy dependency, not that one !

Reply:
smack-legacy is what I mean (the github interface shows only the first line of what I clicked on). Does the code still need smack-legacy?

Reply:
Yes

Reply:
I used that it when I used the MessageEvent class